code/game/objects/effects/spiderwebs.dm

```diff
@@ -65,7 +65,7 @@
 	if(isspider(mover))
 		return TRUE
 	else if(isliving(mover))
-		if(istype(mover.pulledby, /mob/living/basic/giant_spider))
+		if(istype(mover.pulledby, /mob/living/basic/spiderling))
```

Comment:
This doesn't seem right, spiders should be able to haul bodies through webs and conversely it would seem odd if spiderlings even _can_ do that.